addons/goggles/CfgEventHandlers.hpp

```diff
@@ -14,15 +14,15 @@ class Extended_PostInit_EventHandlers {
 class Extended_Killed_EventHandlers {
     class CAManBase {
         class ADDON {
-            killed = QUOTE(_this call FUNC(handleKilled));
+            clientKilled = QUOTE(_this call FUNC(handleKilled));
         };
     };
 };
 
-class Extended_Fired_EventHandlers {
+class Extended_FiredBIS_EventHandlers {
     class CAManBase {
         class ADDON {
-            fired = QUOTE(if (local (_this select 0)) then {_this call FUNC(handleFired)});
+            clientFiredBIS = QUOTE(if (local (_this select 0)) then {_this call FUNC(handleFired)});
```

Comment:
`clientXEH_Event` still runs on all clients, it just removes the EH from dedicated servers

The effective result is something like this:
```
if (local (_this select 0)) then {
    _this call {
        params ["_unit"];
        if (_unit != ACE_player) exitWith {true};
        //CODE
    };
};
```
Right now we are doing `select 0` twice (`local` chceck and later in `param`); So we could remove the local check and I think it should be roughly the same speed?

Personally I like having the XEH lines clean and moving all code to SQF.

But all of this is a more general problem that can be taken care of at a later time.